src/macros.rs

```diff
@@ -530,16 +534,17 @@ macro_rules! service {
         pub struct SyncClient(FutureClient);
 
         impl $crate::client::sync::Connect for SyncClient {
-            fn connect<A>(addr: A, options: $crate::client::Options)
+            fn connect<A>(_addr: A, _opts: $crate::client::Options)
```

Comment:
Oh, I didn't think about that. I guess an underscore suffix would accomplish the same thing.